lapis2/src/main/kotlin/org/genspectrum/lapis/model/VariantQueryCustomListener.kt

```diff
@@ -14,18 +14,30 @@ import VariantQueryParser.NucleotideInsertionQueryContext
 import VariantQueryParser.NucleotideMutationQueryContext
 import VariantQueryParser.OrContext
 import VariantQueryParser.PangolineageQueryContext
+import org.antlr.v4.runtime.RuleContext
 import org.antlr.v4.runtime.tree.ParseTreeListener
+import org.genspectrum.lapis.silo.AminoAcidInsertionContains
 import org.genspectrum.lapis.silo.AminoAcidSymbolEquals
 import org.genspectrum.lapis.silo.And
 import org.genspectrum.lapis.silo.HasAminoAcidMutation
 import org.genspectrum.lapis.silo.HasNucleotideMutation
 import org.genspectrum.lapis.silo.Maybe
 import org.genspectrum.lapis.silo.NOf
 import org.genspectrum.lapis.silo.Not
+import org.genspectrum.lapis.silo.NucleotideInsertionContains
 import org.genspectrum.lapis.silo.NucleotideSymbolEquals
 import org.genspectrum.lapis.silo.Or
 import org.genspectrum.lapis.silo.PangoLineageEquals
 import org.genspectrum.lapis.silo.SiloFilterExpression
+import org.genspectrum.lapis.silo.StringEquals
+
+const val PANGO_LINEAGE_COLUMN = "pangoLineage"
+const val NEXTCLADE_PANGO_LINEAGE_COLUMN = "nextcladePangoLineage"
+const val NEXTSTRAIN_CLADE_COLUMN = "nextstrainClade"
+const val GISAID_CLADE_COLUMN = "gisaidClade"
```

Comment:
Do we have some file, where we can put the custom settings for the covSpectrum Instance? Or also some docu?